tools/image-dev

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
```

Comment:
This is fantastic!!!!

Reply:
Yeah,  give it a try sometime,  under the right conditions it's useful.